### ISSUE TITLE
Measurement Precision

### ISSUE THREAD

**Issue by mcabral5** [mcabral5/Mako-Calibration#3]
This issue is using the same script file 'measure.py'

Camera measurements are given in terms of cm. I have seen examples giving measurements with higher resolution(units in mm or cm with decimals), however, I have not discovered how to alter the code to reflect this. The current resolution is in whole cm. An image is provided below:
<img width="564" alt="image" src="https://user-images.githubusercontent.com/84546893/163257678-02a29d24-581c-4ea0-b84e-e8cb24a08d1e.png">

**Comment by chicostate-mmem** [comment]
If you have those values returning from a function, you can create another function or method to scale that into the dimension we are looking for. Does it make sense?

**Comment by mcabral5** [comment]
> If you have those values returning from a function, you can create another function or method to scale that into the dimension we are looking for. Does it make sense?

Yes, I believe I can do that. I printed out the rotation and position matrices and got the following:
<img width="220" alt="image" src="https://user-images.githubusercontent.com/84546893/163269106-602dd953-c9f9-4f80-b903-d0e2e9b7454e.png">

I will try and scale these numbers to microns or mm and print them.

**Comment by chicostate-mmem** [comment]
Rotation matrix values might be in radians though since the numbers are between 0-3.14 (pi).

**Comment by mcabral5** [comment]
> Rotation matrix values might be in radians though since the numbers are between 0-3.14 (pi).

I believe they are still degrees. The code is using math.degree. 
<img width="673" alt="image" src="https://user-images.githubusercontent.com/84546893/163277261-bb2bdbe4-4575-46a9-9b39-56079cd4f3bd.png">

**Comment by mcabral5** [comment]
Updated frame:
<img width="723" alt="image" src="https://user-images.githubusercontent.com/84546893/163277489-1b13a8da-0580-4d0e-8c3e-bc5e04108156.png">

**Comment by chicostate-mmem** [comment]
Let's find out the base frame and compare the difference in position with the motion of the linear stage.
